src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,5 +14,21 @@ public class Messages {
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_DUPLICATE_FIELDS = "Detected multiple values for the following field%1$s: "
+            + "%2$s.\nOnly one value is allowed for %3$s field%1$s\n";
+
+    /**
+     * Formats the duplicate prefixes into an error message.
+     */
+    public static String getDuplicatePrefixesToMessage(Set<Prefix> duplicatePrefixes) {
```

Comment:
alternative to consider:
```suggestion
    public static String getDuplicatePrefixesToMessage(Prefix... duplicatePrefixes) {
```

Reply:
I kept it intentionally this to avoid ordering of the fields in the message to avoid complications when doing unit test etc. 
It is possible to sort the field within this method but I feel that supplying it with a `Set` will make it cleaner.

Reply:
Instead of creating `Set` objects everywhere this method is called, you can create the `Set` inside the method at the start?

Reply:
Yes, but the code technically is really only called once in production code and the rest of it is called in unit test.

Which is called below.
```java
    /**
     * Throws a {@code ParseException} if any of the prefixes given in {@code prefixes} appeared more than
     * once among the arguments.
     */
    public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseException {
        Set<Prefix> duplicatedPrefixes = new HashSet<>();
        Set<Prefix> prefixSet = Set.of(prefixes);
        argMultimap.forEach((key, list) -> {
            if (list.size() > 1 && prefixSet.contains(key)) {
                duplicatedPrefixes.add(key);
            }
        });

        if (!duplicatedPrefixes.isEmpty()) {
            throw new ParseException(Messages.getDuplicatePrefixesToMessage(duplicatedPrefixes));
        }
    }
```
Converting `getDuplicatePrefixesToMessage()` to take in `Prefix...` or `Prefix[]` will involve calling `duplicatedPrefixes.toArray(Prefix[]::new))` which will cause it to be converted once again into a `Set` object when inside the method. 

This is true unless we are using an fixed-size array for `duplicatedPrefixes` or alternative we can convert the whole process as a java stream and convert it to an array. 

I will experiment processing the above with java stream and see it's readability in the next version.

Reply:
I think the code is cleaner with java stream, I will update the relevant code.

Reply:
BTW, you are doing something very similar in here:
```java
 public void verifyNoDuplicatePrefixesFor(Prefix... prefixes) throws ParseException {
        Set<Prefix> duplicatedPrefixes = new HashSet<>();
        Set<Prefix> prefixSet = Set.of(prefixes);
```